packages/plugin-ext/src/main/browser/plugin-ext-deploy-command.ts

```diff
@@ -76,7 +80,7 @@ export class DeployQuickOpenItem extends QuickOpenItem {
 
     constructor(
         protected readonly name: string,
-        protected readonly pluginServer: PluginServer,
+        protected readonly pluginExtDeployCommandService: PluginExtDeployCommandService,
```

Comment:
If we keep the proposed implementation, can you add it to `changelog` as a breaking change, it breaks downstream applications who extend or rebind `PluginExtDeployCommandService`.